lib/util.js

```diff
@@ -1175,37 +1157,38 @@ function formatMapIterInner(ctx, value, recurseTimes, keys, entries, state) {
   if (remaining > 0) {
     output.push(`... ${remaining} more item${remaining > 1 ? 's' : ''}`);
   }
-  for (i = 0; i < keys.length; i++)
-    output.push(formatProperty(ctx, value, recurseTimes, keys[i], 0));
   return output;
 }
 
-function formatWeakSet(ctx, value, recurseTimes, keys) {
+function formatWeak(ctx) {
```

Comment:
Could this be name `formatWeakCollection`?

Reply:
Done